persistence-modules/spring-boot-persistence-2/src/main/java/com/baeldung/db/indexing/Image.java

```diff
@@ -0,0 +1,62 @@
+package com.baeldung.db.indexing;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Lob;
+
+@Entity
+class Image {
+
+    @Id
+    @GeneratedValue
+    long id;
```

Comment:
It's generally considered better to use a wrapper type (Long) for the id instead of the primitive (long).